turbopack/crates/turbo-tasks/src/macro_helpers.rs

```diff
@@ -199,3 +199,21 @@ macro_rules! inventory_submit {
 /// Exported so the above macro can reference it.
 #[doc(hidden)]
 pub use inventory::submit as inventory_submit_inner;
+
+/// Define a global name for a turbo-tasks value.
+#[cfg(not(rust_analyzer))]
+#[macro_export]
+macro_rules! global_name {
+    ($($item:tt)*) => {
+
+        concat!(env!("CARGO_PKG_NAME"), "@", module_path!(), "::", stringify!($($item)*))
```

Comment:
Nit: Technically these `concat`, `env`, `module_path`, and `stringify` macros should be referred to by an absolute path.